src/xml/merge-multi-xmi.xsl

```diff
@@ -23,16 +23,15 @@
     <!-- List of xmi/xml UML files to be merged into a single file -->
     <!--<xsl:import href="../../src/common/functx-1.0.1-doc.xsl"/>-->
     <xsl:variable name="inputFolder" select="functx:substring-before-last(base-uri(), '/')"/>    
-    <xsl:variable name="xmiFileList" select="collection(concat($inputFolder, '/', '?select=*.xml'))"/>
-    
+    <xsl:variable name="FileList" select="collection(concat($inputFolder, '/', '?select=*.xmi'))  | collection(concat($inputFolder, '/', '?select=*.xml'))"/>
```

Comment:
nice update. 

I hope this does not create issues with the "metada.xml" file in case it is included by mistake (which should probably not happen).